src/CtrlGroup.c

```diff
@@ -53,6 +53,18 @@ const char* Ros_CtrlGroup_GRP_ID_String[] =
     "s24",
 };
 
+CtrlGroup* Ros_CtrlGroup_Ctor()
+{
+    CtrlGroup* grp = (CtrlGroup*)mpMalloc(sizeof(CtrlGroup));
+    bzero(grp, sizeof(CtrlGroup));
+    return grp;
+}
+
+void Ros_CtrlGroup_Dtor(CtrlGroup* grp)
```

Comment:
Should this get merged with `Ros_CtrlGrp_Cleanup` or called in `Ros_Controller_Cleanup`

Reply:
The former does some [rather specific things](https://github.com/Yaskawa-Global/motoros2/blob/316b9993b5d1f1be940ffd46147934261726e596/src/CtrlGroup.c#L236-L242) and doesn't release the memory of the `CtrlGroup` passed in.

We should indeed start using this in `Ros_Controller_Cleanup(..)` [here](https://github.com/Yaskawa-Global/motoros2/blob/316b9993b5d1f1be940ffd46147934261726e596/src/ControllerStatusIO.c#L246). But I would again suggest that's not in scope in this PR.